From: Philip Hands <phil@hands.com>
Date: Mon, 29 Apr 2024 22:37:01 +0200
Subject: Debianise the suggested xtigervncviewer command

Debian uses a different command line, with a lower-case -shared option
for TigerVNC, so make the suggested command one that works for us.
---
 templates/webapi/test/live.html.ep | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/webapi/test/live.html.ep b/templates/webapi/test/live.html.ep
index dfcade0..89ebbbd 100644
--- a/templates/webapi/test/live.html.ep
+++ b/templates/webapi/test/live.html.ep
@@ -46,7 +46,7 @@
         <div class="row">
             <div class="col-sm-12" id="developer-flash-messages">
                 <div id="developer-vnc-notice" class="alert alert-secondary developer-mode-element" role="alert" data-visible-on="isPaused,vncArg">
-                    You might be able to connect to the SUT at <span class="vnc-arg">?</span> via VNC with shared mode enabled (eg. <code>vncviewer <span class="vnc-arg">?</span> -Shared</code> for TigerVNC).
+                    You might be able to connect to the SUT at <span class="vnc-arg">?</span> via VNC with shared mode enabled (eg. <code>xtigervncviewer <span class="vnc-arg">?</span> -shared</code> for TigerVNC).
                 </div>
             </div>
         </div>
